src/NuGet.Core/NuGet.Configuration/PackageSourceMapping/PackageSourceMapping.cs

```diff
@@ -20,34 +22,39 @@ public class PackageSourceMapping
         /// </summary>
         internal IReadOnlyDictionary<string, IReadOnlyList<string>> Patterns { get; }
 
-        private Lazy<SearchTree> SearchTree { get; }
+        private Lazy<SearchTree>? SearchTree { get; }
 
         /// <summary>
         /// Indicate if any packageSource exist in package source mapping section
         /// </summary>
         public bool IsEnabled { get; }
```

Comment:
I can't remember the exact name of the attribute, but there's an attribute something like `NotNullMemberWhen`, which tells the nullable checkers that when this property is `true`, then the referenced member is not null, so you can write code like `if (IsEnabled) { SearchTree.Value.Method(); }` and it won't force you to check the nullability of `SearchTree`. Although, `SearchTree` is private, and it seems that we don't have code here that checks `IsEnabled`, so maybe it's more of an idealistic suggestion than a practical one.